### PR TITLE
[Localization] Using PrBuild Variable

### PR DESCRIPTION
**Pull request by tj-devel709**
Changes to use PrBuild variable as check for running OneLocBuild task instead of checking if Build.SourceBranchName is set to 'merge'.

### CODE REVIEW

File: tools/devops/automation/templates/build/configure.yml [hunk 1]
```diff
@@ -106,9 +107,11 @@ steps:
...
 - task: PublishBuildArtifacts@1
-  condition: eq('$(Build.SourceBranchName)', 'merge')
+  condition: eq('$[PrBuild]', 'True')
   continueOnError: true
   inputs:
     PathtoPublish: '$(Build.ArtifactStagingDirectory)'
     ArtifactName: 'localizationDrop'
     publishLocation: 'Container'
+  env:
+    PrBuild: $(labels.prBuild)
```
Comment (mandel-macaque) [inline tools/devops/automation/templates/build/configure.yml:117]: Not needed and is not doing what you think since it is a env variable for scripts.

**Review by mandel-macaque — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Comment by mandel-macaque** [comment]
@tj-devel709  as alex said, should be easy to do and and () and add the comparison with the branch name and 'main'

File: tools/devops/automation/templates/build/configure.yml [hunk 2]
```diff
@@ -90,7 +90,7 @@ steps:
   workingDirectory: $(Build.SourcesDirectory)\tools\devops 
 
 - task: OneLocBuild@2
-  condition: contains(variables['labels.prBuild'], 'False')
+  condition: and (contains(variables['labels.prBuild'], 'False'), eq('$(Build.SourceBranchName), 'main'))
```
Comment (dalexsoto) [inline tools/devops/automation/templates/build/configure.yml:93]: Almost, check your single quotations on both changes :)

Comment (tj-devel709) [reply tools/devops/automation/templates/build/configure.yml:93]: ahh...

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**